### ISSUE TITLE
Update users table for demo purposes

### ISSUE THREAD

**Issue by amenity** [cityofaustin/atd-data-tech#4716]
We're not going to be demoing the Staff page, so the only place that user data will appear is on the "Assign Team" page of the project creation workflow. 

We want this drop down to have a nice, short list of users:
<img width="437" alt="Screen Shot 2020-12-15 at 9 54 50 AM" src="https://user-images.githubusercontent.com/1463708/102283796-b1c19f00-3ef8-11eb-9c21-ab14b591644b.png">

**Comment by amenity** [comment]
@mddilley -[Here are some users](https://docs.google.com/spreadsheets/d/1_QVFEypulJg1A4hks_AaeS4lY0EEReSb/edit#gid=933265784). 

Let me know if you need more, or additional columns. 🙏

**Comment by mddilley** [comment]
All of these users are now added to staging for the demo. I gave them all a title of `Demo User` to differentiate them from the real staging accounts. Tested that the full length workgroups do appear in the UI for all of them. Closing.

![Screen Shot 2020-12-16 at 12.13.33 PM.png](https://images.zenhubusercontent.com/5cfe7d078509cb6dcbaafdc3/f600f91c-6b23-436c-9052-73a9a4964da1)

![Screen Shot 2020-12-16 at 1.32.18 PM.png](https://images.zenhubusercontent.com/5cfe7d078509cb6dcbaafdc3/eb19355e-0d95-4fc5-91e4-1edfaa2f68a0)
